docs/visual_parity.md

```diff
@@ -0,0 +1,56 @@
+# Visual Parity
+
+Quilla includes a special `XPath` validation state called `VisualParity`. This page describes what `VisualParity` is in detail, how it can be used for UI regression tests, and how it integrates storage plugins to manage the images it uses and creates. Quilla strives to make the handling of auxiliary data (baseline images, etc) needed by VisualParity as simple as possible.
+
+## What is VisualParity?
+
+VisualParity is a validation state that uses previously accepted screenshots generated by Quilla ("baseline images") to compare against screenshots generated when Quilla is run during testing ("treatment images") to determine if the new screenshots match the old ones. By using VisualParity with small sections of a web page, Quilla can give assurances that each section remains the same as it has been from previous runs, allowing users to update only the baseline images related to the work they are doing and protecting the rest of the page from having unexpected visual changes introduced.
```

Comment:
Either small sections or even the whole page of a web page

Reply:
My reasoning for specifying "sections of the web page" instead of whole web page is that I'm a little concerned that using a screenshot of the whole page will inevitably lead to very brittle tests, and if the tests are too brittle it will discourage people from using them